telegram/ext/__init__.py

```diff
@@ -28,17 +28,6 @@
 from .contexttypes import ContextTypes
 from .dispatcher import Dispatcher, DispatcherHandlerStop, run_async
 
-# https://bugs.python.org/issue41451, fixed on 3.7+, doesn't actually remove slots
-# try-except is just  here in case the __init__ is called twice (like in the tests)
-# this block is also the reason for the pylint-ignore at the top of the file
```

Comment:
can you also remove the pylint-ignores?

Reply:
we can also remove all py 3.7 workarounds (if there were more?)

Reply:
where there? I just found one in test_ppersistence and I'm fine with leaving that …